module/surveys/helper.js

```diff
@@ -1867,7 +1887,8 @@ module.exports = class SurveysHelper {
     userId = "",
     token = "",
     appVersion = "",
-    appName = ""
+    appName = "",
+    isTransformationRequired = true
```

Comment:
@jayesh9747  add isTransformationRequired  in function signtaure and give description